modules/installation-supported-aws-machine-types.adoc

```diff
@@ -130,11 +130,41 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 |x
```

Comment:
After https://github.com/openshift/installer/pull/5334 , the current list of bootstrap type is outdated. I believe we need to fix this list too. Let me know wdyt @patrickdillon